config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics-main-upgrades.yaml

```diff
@@ -19,7 +19,7 @@ periodics:
     path_alias: k8s.io/kubernetes
   spec:
     containers:
-    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20221024-d0c013ee2d-1.24
+    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20221024-d0c013ee2d-1.25
```

Comment:
Are you able to validate locally that tests are passing with go 1.19? We've seen occurrences in the past where bumping the go version broke `make verify`

Reply:
yes, I have verified it and other verify target that are noted here: https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2773#issue-1433952946

Reply:
Have you tried it from the main branch as well? just want to make sure the tests won't start failing on other PRs until we merge https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2773

Reply:
hmm if I got your question right, running `make verify` from CAPZ main with go version 1.19 installed locally went fine. 
Also, perhaps I have to strip out go uplift changes from https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2773 and put it in a separate PR for the smoother transition since that was the order of PRs being merged previously:

1. container Image uplift: https://github.com/kubernetes/test-infra/pull/26864 
2. go upliift: https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2487
3. CAPI upliift: https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2431

Reply:
> running make verify from CAPZ main with go version 1.19 installed locally went fine.

that's what I was looking for. We've had issues in the past where the new version in CI broke some stuff due to differences in the new go version.